packages/server/src/core/types.ts

```diff
@@ -59,7 +60,7 @@ type GetInferenceHelpers<
       : TRouterOrProcedure extends AnyProcedure
       ? TType extends 'input'
         ? inferProcedureInput<TRouterOrProcedure>
-        : inferProcedureOutput<TRouterOrProcedure>
+        : inferTransformedProcedureOutput<TRouterOrProcedure>
```

Comment:
This is a breaking change in another way than the other issue.

If you have an integration test or use the ssg helper, the data is unchanged.

I don't have a better idea right now but I am not confident this is the right solution

Reply:
This was actually intentional b/c I didn't want a breaking change

Reply:
I agree - we should not change this. If someone wants the transformed type they can use `inferTransformedProcedureOutput` directly (for now, at least).